plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/internal/macrolayout/KlighdDiagramLayoutConnector.java

```diff
@@ -1151,6 +1206,7 @@ private void handleExcludedEdge(final KEdge edge) {
         final KNode targetNode = edge.getTarget();
         final KPort sourcePort = edge.getSourcePort();
         final KPort targetPort = edge.getTargetPort();
+//        if (targetNode == null && targetPort == null) return;
```

Comment:
I do not get why this is commented? Either remove this or add it to prevent edges with no valid target from being processed.